api/server/services/Config/getCustomConfig.js

```diff
@@ -10,17 +10,7 @@ const getLogStores = require('~/cache/getLogStores');
  * */
 async function getCustomConfig() {
   const cache = getLogStores(CacheKeys.CONFIG_STORE);
-  let customConfig = await cache.get(CacheKeys.CUSTOM_CONFIG);
-
-  if (!customConfig) {
-    customConfig = await loadCustomConfig();
-  }
-
-  if (!customConfig) {
-    return null;
-  }
-
-  return customConfig;
+  return (await cache.get(CacheKeys.CUSTOM_CONFIG)) || (await loadCustomConfig());
```

Comment:
This changes at the logic a little bit and doesn't return null from the function anymore. Are you sure that's not needed?

Reply:
`await loadCustomConfig()` already returns null in some cases and a config object in others. So `(await loadCustomConfig()) || null` is not necessary.